packages/legacy/package.json

```diff
@@ -67,6 +63,7 @@
     "@typescript-eslint/parser": "^5.59.0",
     "@viamrobotics/eslint-config": "^0.0.4",
     "@viamrobotics/prettier-config": "^0.0.1",
+    "@viamrobotics/prime-core": "workspace:../core",
```

Comment:
Oh duh, hadn't considered that we'd added a `workspace` dependency. This makes our usage of `pnpm publish -r` _required_ rather than "nice to have" (to replace `workspace` deps with the actual versions).

Glad you took care of that!

Reply:
I just added this dependency so we could share the theme across packages, but we didn't do that when you first reviewed. It's much better now anyways!